include/mbedtls/mbedtls_config.h

```diff
@@ -1159,6 +1159,7 @@
  * Module:  library/x509_create.c
  *
  * Requires: MBEDTLS_BIGNUM_C, MBEDTLS_OID_C, MBEDTLS_PK_PARSE_C,
+ *           MBEDTLS_MD_C
```

Comment:
It seems that the commit message that introduced this change should be updated because it's not related to what the commit actually implements.